cpu/lpc2387/lpc23xx-iap.c

```diff
@@ -32,9 +32,9 @@
 /* pointer to reserved flash rom section for configuration data */
 __attribute((aligned(256))) char configmem[256] __attribute__((section(".configmem")));
 
-static unsigned int iap_command[5];         // contains parameters for IAP command
-static unsigned int iap_result[2];          // contains results
-typedef void (*IAP)(unsigned int[], unsigned int[]);    // typedefinition for IAP entry function
+static unsigned int iap_command[5];         /* contains parameters for IAP command */
+static unsigned int iap_result[2];          /* contains results */
+typedef void (*IAP)(unsigned int[], unsigned int[]);    /* typedefinition for IAP entry function */
```

Comment:
These comments make the line longer than 80 chars, but I'm not sure if moving them a line up makes it more clear.